src/components/NavBottom.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import {NavLink} from 'react-router-dom';
+
+export default function NavTop() {
+  return (
+    <Nav>
+      <Link to="ScalePage">ScalePage</Link>
+      <Link to="/">FormPage</Link>
+    </Nav>
+  );
+}
+
+const Nav = styled.nav`
```

Comment:
try the `<nav>` html element here for more semantic code and better a11y ;)

Reply:
Hi Julia. Thanks for your effort to help out with some comments. What do you mean with this one? I try to use the styled-components method. Not sure about the nav html?